### ISSUE TITLE
How to disable whitespace trimming?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#251]
```
It's probably not a bug. However XDocReport trims whitespace characters when 
converting odt to PDF format. 
How can I disable that behaviour?

The thing is that I have two words that need to be in the same line: one on the 
left, second on the right.

Also I do not understand while in the PDF document XDoc put "1.1" before my 
header... Is this a result od default settings?

odt and pdf files attached.


```

Original issue reported on code.google.com by `fantasty...@gmail.com` on 23 Apr 2013 at 7:06

Attachments:
- [my_document.odt](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-251/comment-0/my_document.odt)
- [my_document.pdf](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-251/comment-0/my_document.pdf)

**Comment by GoogleCodeExporter** [comment]
```
Many thank's to have attached your odt which have problems. I think it's 2 bugs.
We will see your problem when we will have time.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 23 Apr 2013 at 7:40
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Can't reproduce the problem in the current version 1.0.2
Whitespace is properly inserted. Heading in pdf has no number, so it is also 
correct.
I think I fixed it in version 1.0.0, please use the newest version.
```

Original comment by `abe...@gmail.com` on 11 Jun 2013 at 7:40
- Changed state: **Invalid**
